eslint.config.mjs

```
@@ -8,6 +8,7 @@
 import angular from 'angular-eslint';
 import tseslint from 'typescript-eslint';
 import eslint from '@eslint/js';
+import boundaries from 'eslint-plugin-boundaries';
```

ESLint

[error] 'boundaries' is defined but never used.

(no-unused-vars)